browser.js

```diff
@@ -98,27 +98,33 @@ function changeTab(next) {
 	$(`a[href$="${pages[ret]}"]`).click();
 }
 
+// sets interval to wait for selector to be ready before firing callback
+function waitFor(selectorString) {
+	return new Promise(resolve => {
+		const awaitElement = setInterval(() => {
```

Comment:
Let's also do the check before the interval in case it already exists before the check started, so we don't have to wait the 100ms.